### DISCUSSION TITLE
Update Automation?

### DISCUSSION THREAD

**Discussion by packetdog** [valinet/ExplorerPatcher#2613] [Q&A]
Good day everyone,

Does anyone know if there is a way to script update ExplorerPatcher? I have other software that we script to update, even to go as far as comparing version on the machine to the latest Github release. I see by Googling this that I can use winget to install it apparently, but not sure if that's the best way?

I guess my questions to the community and/or author are:
- Is anyone scripting this update today?
- (to Valinet) Is it possible to replicate what happens when you click "Update program and restart file explorer" in the application, with a script? This link (on my PC) also elevates immediately after clicking it, so for a script I'd want to run this as an admin (or SYSTEM) to start with.
- Do we know if an update needs to run in the user context, Admin, or SYSTEM?

Thanks for any ideas anyone has.
-pd

**Comment by pyrates999** [comment]
Updates require administrative privilege's because symbols need to be downloaded for the current update that was last installed for windows.  And symbols change for every update.

https://github.com/valinet/ExplorerPatcher/wiki/Symbols
